### PR TITLE
#3 - Backend:add versioneye

### PR DESCRIPTION
**Pull request by olegbal**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,16 +4,19 @@ buildscript {
     }
     repositories {
         mavenCentral()
+        jcenter()
     }
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath('org.standardout:gradle-versioneye-plugin:1.4.0')
```
Comment (solairerove) [inline build.gradle]: Version to properties

Comment (solairerove) [reply build.gradle]: Add one style

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# VersionEye project ID added by Gradle VersionEye plugin
+versioneye.organisation=olegbal_orga
+versioneye.api_key=c7c58048fc0a6646af7f
+# VersionEye project ID added by Gradle VersionEye plugin
```
Comment (solairerove) [inline gradle.properties]: What's about dis keys. Are they private?

File: gradle.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+versioneye.organisation=olegbal_orga
```
Comment (vlsidlyarevich) [inline gradle.properties]: can you make it from robanukah organization?

File: build.gradle (hunk 1 above)
Comment (vlsidlyarevich) [inline build.gradle]: it would be better if you separate versions in file

**Review by solairerove — APPROVED** [review]
wasted

**Comment by solairerove** [comment]
closed

**Pull request merged**